qgrid/grid.py

```diff
@@ -214,16 +216,26 @@ class QGridWidget(widgets.DOMWidget):
     df = Instance(pd.DataFrame)
     precision = Integer(6)
     grid_options = Unicode('', sync=True)
-    remote_js = Bool(True)
+    remote_js = Bool(False)
+
+    def __init__(self, *args, **kwargs):
+        """Initialize all variables before building the table."""
+        kwargs.setdefault('grid_options', json.dumps(defaults.grid_options))
```

Comment:
cc @ellisonbg.  Do you know of any projects that are using traitlets in an IPython 3/4 compatible way?  In the old traitlets, we would use `_grid_options_default` here, but I think that API is deprecated and/or removed in the latest `traitlets` release?